### ISSUE TITLE
RPI 4 install still not working but issue was closed

### ISSUE THREAD

**Issue by shiftylilbastrd** [sourceryltd/makerhub-releases#7]
https://github.com/sourceryltd/makerhub-releases/issues/5#issuecomment-2027804985

**Comment by shiftylilbastrd** [comment]
As shown at the link above to the previous issue Fuse 3 is already installed but produces an error.
I installed Fuse 2 `apt install libfuse2`. 
Then run `./MakerHub-1.3.1-beta1-arm64.AppImage` and get
```
Current directory: /tmp/.mount_MakerHtwKDq0/resources/app
Linux OS Detected
runonce.txt not found - patching now!
Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
null
null
Error: /tmp/.mount_MakerHtwKDq0/resources/app/node_modules/@serialport/parser-byte-length/dist/index.js:11
    length;
          ^

SyntaxError: Unexpected token ;
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:660:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/tmp/.mount_MakerHtwKDq0/resources/app/node_modules/serialport/dist/index.js:17:14)
    at Object.<anonymous> (/tmp/.mount_MakerHtwKDq0/resources/app/node_modules/serialport/dist/index.js:30:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
~~~

**Comment by kevinjohnson123** [comment]
Check the releases section for the    ARM v7  AppImage.  I can confirm it does load on a  Raspberry Pi 3 B+.   I had to increase the swap file size from 100 MB to  1GB in order for it to build, i haven't tested if 1GB of Swap is needed to run Makerhub on the Pi3  or if less will work.

you also might need to install fuse.  Depending on which OS and version of Rasapian you're using fuse might already be included or might not be.  I wasn't sure which version of fuse the AppImage (Electron Builder actually)  needs so i installed  both fuse2 and fuse3 to my development system.

sudo apt-get install libfuse2 fuse-utils python-fuse
sudo apt-get install libfuse3


when i launched MakerHub  on the Pi3  it will give some error about the workspace being corrupt.  That's normal,  it just means it can't find an already  existing  .makerhub   configuration file and will auto-generate one.   Aside from that, i got a weird error message several times in a row saying that  "the server has crashed",  even though it didn't.   I just kept clicking  "ok"  for it to restart the server  and after about 3 times it stopped complaining.   I launched Makerhub from the console but i didn't see any error messages.  Might possibly just be a low-memory problem of the  Pi3  having only about 800MB free.  This might not be a problem on a Raspberry Pi4 with 4GB of RAM, though i can't confirm because i don't currently have a Raspberry Pi 4.  
If the  arm v7L  AppImage doesn't launch on a Raspberry Pi4  let me know and i'll go pickup a Pi4 and rebuild on that hardware directly.
